checkov/common/vcs/base_vcs_dal.py

```diff
@@ -58,14 +58,14 @@ def setup_http_manager(self, ca_certificate: str | None = None) -> None:
             except KeyError:
                 self.http = urllib3.PoolManager()
 
-    def _request(self, endpoint: str) -> dict[str, Any] | None:
+    def _request(self, endpoint: str, allowed_status_codes: list[int]) -> dict[str, Any] | None:
```

Comment:
i think you need to init it with default value (but not inside the func signature). some other classes (like gitlab) also call this function